packages/editor/src/plugins/image/components/image-selection-screen.tsx

```diff
@@ -19,6 +21,8 @@ export function ImageSelectionScreen(
   const { state, urlInputRef, setIsAButtonFocused } = props
   const { src, licence } = state
 
+  // const parentType = selectParentPluginType(store.getState(), pluginId)
```

Comment:
Idea being: If the parentType is ImageGallery, then the Upload button can get passed a prop "multiple"

Reply:
(Perhaps not necessary, need to investigate the config and UploadHandler)

Reply:
To save you some time on investigation: https://github.com/serlo/frontend/pull/4042